sth_app/lib/main.dart

```diff
@@ -9,17 +10,9 @@ class SthApp extends StatelessWidget {
   const SthApp({super.key});
   @override
   Widget build(BuildContext context) {
-    return MaterialApp(
-      title: 'Your App',
-      // Define initial route
-      initialRoute: '/homescreen',
-      // Define routes
-      routes: {
-        '/homescreen': (context) => const HomeScreen(),
-        // Add more routes as needed
-        // Example:
-        // '/secondPage': (context) => SecondPage(),
-      },
-    );
+    return MaterialApp(title: 'STH App', debugShowCheckedModeBanner: false, initialRoute: '/homescreen', routes: {
+      '/homescreen': (context) => const HomeScreen(),
+      '/chatScreen': (context) => const chatScreen(),
```

Comment:
Can we write '/chatscreen': (context) => const ChatScreen(), ??

Reply:
yes